### ISSUE TITLE
添加量化训练后显存oom

### ISSUE THREAD

**Issue by koibiki** [aovoc/nnieqat-pytorch#13]
你好，我添加nnieqat量化训练后，每轮训练显存不释放，显存递增后oom

**Comment by koibiki** [comment]
我的环境是 pytorch 1.6  观察是loss backward后，显存不释放。

**Comment by koibiki** [comment]
通过注释，基本确定是激活图量化那块导致的。我对量化也不太了解，希望作者能有空看一下

**Comment by koibiki** [comment]
解决了，不知道是不是只有我遇到了这个问题，我在　_quantizing_activation　这个函数，将　module.activation_max_value = torch.max(torch.max(torch.abs(output)), module.activation_max_value.to(output_type))　改成了　module.activation_max_value = torch.max(torch.max(torch.abs(output.detach())), module.activation_max_value.to(output_type))

**Comment by sherylwang** [comment]
I've met the same problem, solved by your provided method. Many thanks @koibiki
